dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraph.java

```diff
@@ -115,16 +120,29 @@ void addPath(DependencyPath path) {
     return result;
   }
 
-  /**
-   * @return a mutable copy of the paths in this graph, usually in breadth first order
-   */
+  /** Returns a mutable copy of the paths in this graph, usually in breadth first order. */
   public List<DependencyPath> list() {
     return new ArrayList<>(graph);
   }
 
   /**
-   * @return all paths to the specified artifact
+   * Returns dependency path of the root node.
+   *
+   * @throws IllegalStateException if the graph is empty
    */
+  public DependencyPath getRootPath() {
+    if (graph.isEmpty()) {
+      throw new IllegalStateException("The graph is empty");
+    }
+    return graph.get(0);
+  }
+
+  /** Returns the dependency paths of the children of the node at {@code dependencyPath}. */
```

Comment:
This comment could be clearer. Is this the paths from the root to what exactly? What are the child paths of a dependency path?

Reply:
Updated as `Returns dependency paths from the root to the dependencies of the node at {@code  dependencyPath}.`.

I thought renaming this to `getDependencyDependencyPaths` but this verbosity does not give clarity. Keeping `getChildPaths`.